config/stack_ufs_s2s.yaml

```diff
@@ -0,0 +1,242 @@
+cmake:
```

Comment:
Do we need `stack_ufs_s2s.yaml` and `stack_ufs_s2s_model.yaml` both?
How do they differ from `stack_noaa.yaml` or `stack_custom.yaml`.

Reply:
I had thought I had removed stack_ufs_s2s.yaml.  The main difference between stack_noaa.yaml and the stack_ufs_s2s_model.yaml are that it doesn't build eigen, ecbuild, eckit, fckit, or atlas as none of those are required.  I was trying to minimize the size of the subsequent container.